### DISCUSSION TITLE
Blog designs?

### DISCUSSION THREAD

**Discussion by stinhambo** [twbs/bootstrap#31157] [General]
Will you be providing layouts for blogs such as the full article, cards for homepage and blog main page with category filters?

Something along these lines -

http://avehtml.liquid-themes.com/blog-split.html
http://avehtml.liquid-themes.com/blog-single.html

**Comment by florianlacreuse** [comment]
See blog example: https://getbootstrap.com/docs/5.0/examples/blog/
